src/NearOrg/Notifications/NotificationButton.jsx

```diff
@@ -58,36 +60,49 @@ const Button = ({ count }) => {
   );
 };
 
+const getNotificationsCountQuery = (blockHeight) => `
+  query NotificationsLeftCountQuery {
+    count: dataplatform_near_notifications_notifications_aggregate(where: {receiver: {_eq: "${accountId}"}, blockHeight: {_gt: ${blockHeight}}}, distinct_on: [blockHeight, initiatedBy]) {
+      aggregate {
+        count
+      }
+    }
+  }
+`;
+
 useEffect(() => {
-  const checkLastBlockHeight = setInterval(() => {
+  const checkNotificationsCountPeriod = setInterval(() => {
+    const shouldFallback = Storage.get("notifications:shouldFallback", notificationFeedSrc);
```

Comment:
Is the 2nd arg a default?

Reply:
It's actually the key of a different widget where the data is stored: https://thewiki.near.page/near.social_vm